DataTool/ToolLogic/Dump/DumpFlags.cs

```diff
@@ -1,5 +0,0 @@
-﻿using DataTool.ToolLogic.List;
-
-namespace DataTool.ToolLogic.Dump {
-    public class DumpFlags : ListFlags { }
```

Comment:
this existed to make it a seperate section in the help

Reply:
5Head